spec/lib/mcb/commands/providers/withdraw_courses.rb

```diff
@@ -0,0 +1,34 @@
+require "mcb_helper"
```

Comment:
Filename should be `withdraw_courses_spec.rb`

Reply:
yea caught it 👍  thanks though